lib/define.h

```diff
@@ -174,6 +174,7 @@ const double dmax = numeric_limits<double>::max();
 static const double XYZ_w[3] = {0.952646074569846, 1.0, 1.00882518435159};
 static const double d50[3] = {0.9642, 1.0000, 0.8250};
 static const double d60[3] = {0.952646074569846, 1.0000, 1.00882518435159};
+static const double d65[3] = {0.9547, 1.0000, 1.0883};
```

Comment:
The correct XYZ values for d65 should be [0.95047,1.00000,1.08883]

![d65](https://user-images.githubusercontent.com/25424489/42838113-bfa8059c-89b4-11e8-9989-d22815614efe.PNG)